### PR TITLE
che #15506 Adding RELEASE.md file with instructions for Major / Minor and Service / Bugfix  releases

### PR DESCRIPTION
**Pull request by ibuziuk**
### What does this PR do?
Adding RELEASE.md file with instructions for Major / Minor and Service / Bugfix  releases

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15506

### CODE REVIEW

**Comment by ibuziuk** [comment]
@nickboldt please review

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Major / Minor Release
+
+- create a branch for the release e.g. `7.6.x`
+- provide a [PR](https://github.com/eclipse/che-devfile-registry/pull/171) with bumping the [VERSION](https://github.com/eclipse/che-devfile-registry/blob/master/VERSION) file to the `7.6.x` branch
+- [![Release Build Status](https://ci.centos.org/buildStatus/icon?subject=release&job=devtools-che-devfile-registry-release/)](https://ci.centos.org/job/devtools-che-devfile-registry-release/) CI is triggered based on the changes in the [`release`](https://github.com/eclipse/che-devfile-registry/tree/release) branch (not `7.6.x`).
+
+In order to trigger the CI once the [PR](https://github.com/eclipse/che-devfile-registry/pull/171) is merged to the `7.6.x` one needs to:
+
+```
+ git fetch origin 7.6.x:7.6.x
```
Comment (nickboldt) [inline RELEASE.md]: should this just be `git fetch origin 7.6.x` ?

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ git checkout 7.6.x
+ git branch release -f 
+ git push origin release -f
+```
```
Comment (nickboldt) [inline RELEASE.md]: Is there a che-bot user that can:
* create 7.a.x branch,
* force push to  release branch, and 
* create 7.a.0 tag ?

File: RELEASE.md (hunk 1 above)
Comment (ibuziuk) [reply RELEASE.md]: no, since you need to checkout to `7.6.x`

File: RELEASE.md (hunk 2 above)
Comment (ibuziuk) [reply RELEASE.md]: I think it is a separate discussion related to automation

**Pull request merged**